text/java/0000-rename-open-liberty-buildpack.md

```diff
@@ -0,0 +1,41 @@
+# Rename open liberty buildpack to liberty
+
+## Summary
+
+The `open-liberty` buildpack can contribute either the Open Liberty or Websphere Liberty runtime.  To avoid confusion the buildpack should simply be named `liberty`.
```

Comment:
```suggestion
The `open-liberty` buildpack can utilize either the Open Liberty or Websphere Liberty runtime.  To avoid confusion the buildpack should simply be named `liberty`.
```

This might be splitting hairs, but the open-liberty buildpack can only contribute dependencies for Open Liberty. It can however work with and configure Websphere Liberty runtime, if it's provided through a base image. 

I just want to avoid overpromising if someone reads this and isn't totally familiar with the capabilities in the buildpack. I think this word change more accurately represents what the buildpack does today.

Reply:
Good point, at some point we can add websphere liberty to the buildpack directly.